cilium-dbg/cmd/post_uninstall_cleanup.go

```diff
@@ -514,19 +504,6 @@ func removeRoutesAndLinks(routes map[int]netlink.Route, links map[int]netlink.Li
 	return nil
 }
 
-func removeNamedNetNSs(netNSs []string) error {
```

Comment:
Why is this being removed? I agree that this is probably good to remove, but removing it should probably be its own commit, just for clarity's sake.

Reply:
This removed the pinned netns by name - we are not pinning this anymore so we do not need to remove the pin/do cleanup.